nsv13/code/modules/clothing/custom_clothes.dm

```diff
@@ -224,10 +220,7 @@
 	item_state = "pilot"
 	item_color = "pilot"
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/pilot
-	armor = list("melee" = 40, "bullet" = 50, "laser" = 50, "energy" = 25, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 100, "acid" = 100)
-	resistance_flags = FIRE_PROOF | ACID_PROOF
-	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT //this needed to be added a long fucking time ago
+	armor = list("melee" = 20, "bullet" = 12, "laser" = 10, "energy" = 10, "bomb" = 60, "bio" = 100, "rad" = 50, "fire" = 90, "acid" = 75)
```

Comment:
Consider toning down the bomb armour while you are there.

Reply:
Yeah originally I still kept it cause they work with torps and missles, but I realize that's a stupid reasons as the only time torps explode is when you're the one shooting them. I removed it now though.

Reply:
Yeah the overmap stuff doesn't use those flags.